### PR TITLE
feat: Add a random quote generator.

### PR DESCRIPTION
**Pull request by adithyanmadhi**
# Add a random quote generator.
Screenshot 2024-02-02 034349
# Jira link
https://mahendrankannan.atlassian.net/browse/ON-29

### CODE REVIEW

File: web-application/random-quote-generator/desktop.ini
Comment (gkmahendran09) [inline web-application/random-quote-generator/desktop.ini:1]: delete this file

File: web-application/random-quote-generator/styles.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    font-family: Arial, sans-serif;
+    background-color: #102f00;
+    margin: 0;
+    padding: 0;
+}
+
+.container {
+    max-width: 600px;
+    margin: 50px auto;
+    background-color: #ec0b0b;
```
Comment (gkmahendran09) [inline web-application/random-quote-generator/styles.css:11]: Try to use web safe colors and make the web page more presentable.
https://html.com/blog/design/html-colors/

**Review by gkmahendran09 — CHANGES_REQUESTED** [review]

File: web-application/random-quote-generator/styles.css (hunk 1 above)
Comment (adithyanmadhi) [reply web-application/random-quote-generator/styles.css:11]: Changed the color combination.

Comment (adithyanmadhi) [reply web-application/random-quote-generator/styles.css:11]: Screenshot 2024-02-05 004317

**Review by gkmahendran09 — APPROVED** [review]

**Pull request merged**